### ISSUE TITLE
Zookeeper should pick up fields from annotated @property definitions

### ISSUE THREAD

**Issue by AdamHillier** [larq/zookeeper#88]
The work-around for now is to do:

```python
field_name: SomeType

@property
def field_name(self):
    return ...
```

**Comment by jneeven** [comment]
A nice solution for this would be to have some `@configurable_property` decorator (as suggested by @lgeiger )

**Comment by AdamHillier** [comment]
This has been resolved by #109.

`Field` can be used as a decorator like so:

```python
@Field
def name(self) -> int
    return self.a * 2
```
